src/clr-angular/layout/vertical-nav/vertical-nav-link.ts

```diff
@@ -19,10 +19,12 @@ import {VerticalNavGroupService} from "./providers/vertical-nav-group.service";
 export class ClrVerticalNavLink {
     constructor(@Optional() private _navGroupService: VerticalNavGroupService) {}
 
+    /* tslint:disable */
     @HostListener("click")
     private expandParentNavGroup(): void {
```

Comment:
Is TSLint complaining because this private method isn't used anywhere, and doesn't know about `@HostListener`? This method could very well be public, and on top of that I'm not 100% sure that Angular is happy about private `@HostListener`s (maybe the methods are called from another class).

Reply:
You are right, tslint did not like it being private. Will update.